shell/list/catalog.cattle.io.clusterrepo.vue

```diff
@@ -1,90 +1,36 @@
 <script>
-import { mapGetters } from 'vuex';
 import ResourceTable from '@shell/components/ResourceTable';
-import Masthead from '@shell/components/ResourceList/Masthead';
-import { Banner } from '@components/Banner';
-import { HIDE_DESC, mapPref } from '@shell/store/prefs';
-import { addObject } from '@shell/utils/array';
-
 export default {
   name:       'ListClusterReposApps',
-  components: {
-    Banner,
-    Masthead,
-    ResourceTable
-  },
-
-  props: {
+  components: { ResourceTable },
+  props:      {
     resource: {
       type:     String,
       required: true,
     },
-
     schema: {
       type:     Object,
       required: true,
     },
-
     rows: {
       type:     Array,
       required: true,
     },
   },
-
   computed: {
-    ...mapGetters(['currentCluster']),
-    hideDescriptions: mapPref(HIDE_DESC),
-
-    typeDescriptionKey() {
-      // Show a different message to cover support for RKE templates in the local cluster
-      // (no current cluster means catalog requests default to local)
-      const key = !this.currentCluster || this.currentCluster.isLocal ? 'typeDescription."catalog.cattle.io.clusterrepo.local"' : 'typeDescription."catalog.cattle.io.clusterrepo"';
-
-      if ( this.hideDescriptions.includes(this.resource) || this.hideDescriptions.includes('ALL') ) {
-        return false;
-      }
-
-      if ( this.$store.getters['i18n/exists'](key) ) {
-        return key;
-      }
-
-      return false;
+    loading() {
+      return this.rows.length ? false : this.$fetchState?.pending;
     }
   },
-
-  methods: {
-    hideTypeDescription() {
-      const neu = this.hideDescriptions.slice();
-
-      addObject(neu, this.resource);
-
-      this.hideDescriptions = neu;
-    },
-  }
 };
 </script>
 
 <template>
   <div>
-    <Masthead
-      :schema="schema"
-      :resource="resource"
-    >
-      <template #typeDescription>
```

Comment:
It has been moved to `components/TypeDescription` https://github.com/aalves08/dashboard/blob/6890-enable-incremental-loading-to-all-list-views/shell/components/TypeDescription.vue#L25-L29. 

The goal was to remove completely this custom view which only existed because of this logic but I have found out it's needed to do a check on our CI/CD pipelines...